### ISSUE TITLE
Flexible colours configuration

### ISSUE THREAD

**Issue by mintty** [OpenBoard-org/OpenBoard#303]
There are 5 colour selectors. Their colours can be configured (although it's not obvious to find how) but apparently they cannot be extended. I'd like to have more selectors handy.

**Comment by kaamui** [comment]
hello,

I'll try to find time after the migration to make documentation on the several options of configurations available. For the 5 selectors, you are not the first asking for, but actually, due to our limited resources, we won't be able to prioritize on it soon.

If it can help, you have a color picker available in the applications that behaves as a tool, so you can constantly change the color of your pen.

**Comment by mintty** [comment]
> If it can help, you have a color picker available in the applications that behaves as a tool, so you can constantly change the color of your pen.

I need to find and click "OpenBoard" - "Settings" - "Pen" before I can configure colours. I didn't see an easier/quick option.

**Comment by ChrisPHL** [comment]
Hi everyone,

I really would like that feature improvement, too.
The color selection needs to be normal color picker combined with e.g. 8 buttons for selection of the latest chosen colors or something like that.

As a workaround so far, can I select _any_ color somehow?

kind regards

Chris

**Comment by kaamui** [comment]
yes by using the color picker in the applications (on the library, the right palette)

![image](https://user-images.githubusercontent.com/11720903/235086165-f546f1d3-dce9-4443-8d4f-3b4d6a369a78.png)

**Comment by kaamui** [comment]
Also note that a feature similar to what is asked in this ticket will be integrated to 1.7.1 (more colors+easier to change)

**Comment by kaamui** [comment]
Now planned for 1.8
